pjsip/include/pjsua2/media.hpp

```diff
@@ -1938,9 +1938,9 @@ struct VideoPreviewOpParam {
     unsigned                windowFlags;
 
     /**
-     * Media format. If left unitialized, this parameter will not be used.
+     * Media format video. If left unitialized, this parameter will not be used.
```

Comment:
- s/unitialized/uninitialized
- rather OOT, while here, perhaps update the docs for more detail info, e.g:
   - if uninitialized, capture device will be opened using PJMEDIA wrapper default format, e.g: on Android it is ..., on iOS it is .
   - if the preview is already opened, this format setting will be ignored (or is it?).